apps/admin-page/src/app/pages/newsletters/newsletters.component.ts

```diff
@@ -34,7 +34,7 @@ export class NewslettersComponent {
     this.isLoading = true;
     this.newsletterService.getNewsletters().subscribe({
       next: (response: INewsletter[]) => {
-        this.newsletterList = response;
+        this.newsletterList = response.sort((a, b) => a.createdAt.localeCompare(b.createdAt)).reverse();
```

Comment:
This has unecesary logic. You can accomplish the same without the reverse by switching around the a and b after the arrow. 

b.createdAt.localCompare(a.createdAt)